hooks/uesPrompt.ts

```diff
@@ -31,15 +31,18 @@ const usePrompt = () => {
 
   const onActive = () => {
     const promptValue = promptOptions[prompt].value;
-    setName(promptValue);
-    setTimeout(() => {
-      promptRef.current.setSelectionRange(
-        promptValue.length,
-        promptValue.length,
-      );
-    }, 100);
+
+    if (!name) {
+      setName(promptValue);
+      setTimeout(() => {
+        promptRef.current.setSelectionRange(
+          promptValue.length,
+          promptValue.length,
+        );
+      }, 100);
+    }
+
     clearInterval(timer);
-    if (name) return;
```

Comment:
I'd prefer to `exit` early instead of nesting logic inside an `if` statement.

Reply:
- actual: nested logic inside `if(!name)`
- required: return early `if (name) return` prior to setName being called